website/src/data/users.tsx

```diff
@@ -574,6 +574,17 @@ const Users: User[] = [
     source: 'https://github.com/sabbour/aks-app-template',
     tags: ['bicep', 'aks', 'kubernetes', 'keda', 'grafana', 'prometheus']
   },
+  {
+    title: 'React Component Toolkit with ChatGPT and OpenAI ',
+    description:
+      'A sample demo for building and testing react components and includes a set of unique features including AI component generation and automatic conversion to Azure APIM Widgets',
+    preview: require('./images/react-component-toolkit-openai.png'),
+    website: 'https://github.com/Azure-Samples',
+    author: 'Jenny Chen',
+    source: 'https://github.com/Azure-Samples/react-component-toolkit-openai-demo',
```

Comment:
Not related to this PR but can you add

```
metadata:
	template: react-toolkit-azd-template@0.0.1
```

to your azure.yaml https://github.com/Azure-Samples/react-component-toolkit-openai-demo/blob/99556ac0d4a91900ca5cd7b69db57fe984696dea/azure.yaml#L4?

This will allow us to look at deployment telemetry via azd.